service/http.directory/src/test/scripts/org/kaazing/gateway/service/http/directory/get.index.check.whole.response.rpt

```diff
@@ -20,37 +20,13 @@
 #
 
 # get.index.and.check.whole.response
-connect tcp://localhost:8000
+connect http://localhost:8000/index.html
 connected
 
-write "GET /index.html HTTP/1.1\r\n"
-write "Host: localhost:8000\r\n"
-write "User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.6; rv:8.0) Gecko/20100101 Firefox/8.0\r\n"
-write "Accept: text/html, application/xhtml+xml, application/xml;q=0.9,*/*;q=0.8\r\n\r\n"
-write notify BARRIER
+write method"GET"
 
 #We need to wait for all the data to be written first
-read await BARRIER
-read "HTTP/1.1 200 OK\r\n"
-read "Content-Length: 119\r\n"
-read "Content-Type: text/html\r\n"
-read /Date: .*/ "\r\n"
-read /ETag: .*/ "\r\n"
-read /Expires: .*/ "\r\n"
-read /Last-Modified: .*/ "\r\n"
-read "Server: Kaazing Gateway\r\n"
-read "\r\n"
-read "<!DOCTYPE html>\n"
-read "<html>\n"
-read "<head> \n"
-read " <meta charset=\"utf-8\">\n"
-read "</head>\n"
-read "\n"
-read "<body>\n"
-read " <h1> THIS PAGE EXISTS!! </h1>\n"
-read "</body>\n"
-read "\n"
-read "</html>\n"
+read status "200" "OK"
```

Comment:
Judging by the name of this test, the test is checking the whole body of the response, so we need to include that in this test

ex. read "<!DOCTYPE html>  \r\n" ....